packages/xo-server-web-hooks/src/index.js

```diff
@@ -6,13 +6,20 @@ function handleHook(type, data) {
   const hooks = this._hooks[data.method]?.[type]
   if (hooks !== undefined) {
     return Promise.all(
-      hooks.map(({ url }) =>
-        this._makeRequest(url, type, data).catch(error => {
-          log.error('web hook failed', {
-            error,
-            webHook: { ...data, url, type },
-          })
-        })
+      hooks.map(({ url, response }) =>
+        response && type === 'pre'
+          ? this._waitForResponse(url, type, data).catch(error => {
+              log.error("web hook didn't receive response from server", {
```

Comment:
It's the same kind of error as the other one, we don't specifically know what happened here either in case of error.